st2common/st2common/log.py

```diff
@@ -139,7 +138,8 @@ def setup(config_file, disable_existing_loggers=False):
                                   disable_existing_loggers=disable_existing_loggers)
         handlers = logging.getLoggerClass().manager.root.handlers
         _add_exclusion_filters(handlers)
```

Comment:
Looks like `excludes` should be passed to this function.

Using keyword arguments would also be a good idea :P